.github/workflows/CI.yml

```diff
@@ -139,6 +140,33 @@ jobs:
           ARCH=x86_64 DEST=build/packages/linux/amd64 tools/packaging/linux/create_rpm.sh
           ARCH=aarch64 DEST=build/packages/linux/arm64 tools/packaging/linux/create_rpm.sh
 
+      - name: Download Package Signing GPG key
```

Comment:
shall we sign deb with gpg in the near future? if yes, it will be better to put signing as a separate job.

Reply:
The reason not to have the GPG signing key operations as a separate job is to minimize the locations where the key has been stored. If we add it as a separate job, we have to copy the key to cache and recover it from a different job, which added another place where the key had been stored, adding risk of leaking the signing key.

Reply:
can't we get the key in that separate job?

Reply:
Yes, that's not a bad idea, I'll look into it to see how this could be optimized, especially when signing packages for other platforms are considered.